packages/maskbook/src/plugins/RedPacket/SNSAdaptor/IconURL.ts

```diff
@@ -0,0 +1,9 @@
+export type IconTypes = 'presentDai' | 'presentDefault' | 'presentOkb' | 'erc20Token' | 'erc721Token'
+
+export const IconURLs: Readonly<Record<IconTypes, string>> = {
```

Comment:
why not `as const`?

```
export type IconTypes = keyof typeof IconURLs

export const IconURLs = {} as const
```

see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions